### DISCUSSION TITLE
Add an operating system field

### DISCUSSION THREAD

**Discussion by musicpanda** [PrestaShop/PrestaShop#34807] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

This form contains fields for PHP and PS versions. However, not one for the operating system. I believe that is a mistake. Many problems are Windows specific. And I suspect that other operating systems such as IIS have their share too. This lack leads to errors that cannot be reproduced.

### Expected behavior

An "operating system" field added

### Steps to reproduce

happens always

### PrestaShop version(s) where the bug happened

8.1

### PHP version(s) where the bug happened

7.4

**Comment by kpodemski** [comment]
I'm not sure about this. Windows-specific errors are a bit tricky, you don't run PrestaShop in production on Windows, very often is a problem of misconfiguration. The problem? Configuration differs, it's different if you use Laragon, it's different if you use Xampp.

Another problem? We have maybe two people using Windows system, reproducing issues will be hard.

**Comment by musicpanda** [comment]
You don't run it in production. But quite a few people run a shadow copy of their shop under Windows for experimentation and development.

The form asks for PHP version and PS version because they are an important factor in determining the cause of an error. But so is operating system. Of course it is not always relevant, but when a problem cannot be reproduced on Linux it should be taken into account as possible cause of the problem.

Also asking for subsystem, like Windows/Xampp might be a good idea.

**Comment by paulnoelcholot** [comment]
Hello @musicpanda,

I can't understand your problem. Could you explain it to me in more detail and above all give me steps to reproduce?

Thanks for your feedback!

**Comment by matks** [comment]
@paulnoelcholot I think @musicpanda wants to add a new field to this form https://github.com/PrestaShop/PrestaShop/issues/new?assignees=&labels=Bug%2CNew&projects=&template=1_bug_report.yml 😉

**Reply by MatShir** [reply]
or a github tag to sort issue ?

**Reply by kpodemski** [reply]
first we need to know the operation system if we want to later add a corresponding label

**Reply by MatShir** [reply]
True !

**Comment by kpodemski** [comment]
I converted this to Discussion to avoid confusion.
